nise/__init__.py

```diff
@@ -1,2 +1,2 @@
-__version__ = "2.5.6"
+__version__ = "2.5.7"
```

Comment:
```suggestion
__version__ = "2.6.0"
```

Since this is new functionality, let's uptick this.